db/schema.rb

```diff
@@ -175,6 +174,12 @@
     t.index ["brand_id"], name: "index_twitter_trackers_on_brand_id", unique: true, using: :btree
   end
 
+  create_table "user", force: :cascade do |t|
```

Comment:
Think it's fine as it is. Not sure if this was added from me having to upgrade my postgres and recreate db/tables but yea.

http://stackoverflow.com/questions/27697119/what-is-cascade-in-rails-schema-rb-and-where-did-it-come-from